app/support/auto_expire_reservation.rb

```diff
@@ -14,11 +14,15 @@ def order_details
     purchased_active_order_details
   end
 
+  def earliest_allowed_time
+    Settings.reservations.timeout_period.seconds.ago
```

Comment:
Hmm, interesting. Looks like ERB will convert end up doing a `to_i` or `to_s` so `12.hours` gets rendered into the YAML as some primitive value. So `.seconds.ago` is fine. For the review period, we use `Time.zone.now + Settings.billing.review_period`, but I would lean towards `ago` going forward.